src/migration/1723118739478-SourceCategory.ts

```diff
@@ -0,0 +1,39 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceCategory1723118739478 implements MigrationInterface {
+  name = 'SourceCategory1723118739478';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "source_category" ("id" text NOT NULL, "value" text NOT NULL, "enabled" boolean NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "UQ_f130e0bac7dea92c2a4084c2f89" UNIQUE ("value"), CONSTRAINT "PK_21e4d5359f2a23fd10053f516e9" PRIMARY KEY ("id"))`,
+    );
+    await queryRunner.query(`ALTER TABLE "source" ADD "categoryId" text`);
```

Comment:
It cant be set by user? Since we dont validate just to sanity check if we need to at least do some length validation.

Reply:
These categories can only be set by us. I am cool to add a max length, what do you think of 64 characters?

Reply:
If its only by use then its fine 👍